src/sentry/conf/server.py

```diff
@@ -1181,6 +1181,8 @@ def create_partitioned_queues(name):
     # Enable functionality for attaching  minidumps to events and displaying
     # then in the group UI.
     "projects:minidump": True,
+    # Backend support for supporting performance issue: endpoints, serializers, etc.
+    "projects:performance-issue-details-backend": False,
```

Comment:
Did we want to split these out too?

Reply:
Yeah, I just rebased and this should be gone now.